components/webui/imports/ui/SearchView/SearchView.jsx

```diff
@@ -98,13 +107,19 @@ const SearchView = () => {
             ],
         };
 
-        // NOTE: Although we publish and subscribe using the name
-        // `Meteor.settings.public.SearchResultsCollectionName`, the rows are still returned in the
-        // job-specific collection (e.g., "1"); this is because on the server, we're returning a
-        // cursor from the job-specific collection and Meteor creates a collection with the same
-        // name on the client rather than returning the rows in a collection with the published
-        // name.
-        return dbRef.current.getOrCreateCollection(jobId).find({}, findOptions).fetch();
+        if (SEARCH_SIGNAL.RESP_DONE !== resultsMetadata.lastSignal) {
+            // avoid refreshing estimatedNumResults after job is DONE
+            // since the count would have been already available in
+            // `resultsMetadata.numTotalResults`
```

Comment:
```suggestion
            // Only refresh estimatedNumResults if the job isn't DONE;
            // otherwise the count would already be available in
            // `resultsMetadata.numTotalResults`
```